bitcoin/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# 0.33.0-alpha - TODO: Set date
+
+- Enforce script size limit when hashing scripts [#2794](https://github.com/rust-bitcoin/rust-bitcoin/pull/2794)
+
+## Upgrade notes
+
+- `MAX_SCRIPT_ELEMENT_SIZE` was removed in [#2794](https://github.com/rust-bitcoin/rust-bitcoin/pull/2794)
+  you may want to use `MAX_REDEEM_SCRIPT_SIZE` or perhaps `MAX_STACK_ELEMENT_SIZE` (see `bitcoin/src/blockdata/constants`).
+- `bitcoin::ecdsa::Error` was replaced by `ecdsa::DecodeError` and `ecdsa::ParseSignatureError`
```

Comment:
```suggestion
- `bitcoin::ecdsa::Error` was replaced by `bitcoin::ecdsa::DecodeError` and `bitcoin::ecdsa::ParseSignatureError`
```
I think we're missing this no?

Reply:
or
```suggestion
- `ecdsa::Error` was replaced by `ecdsa::DecodeError` and `ecdsa::ParseSignatureError`
```

Reply:
Oh I thought that bit was obvious and was trying to keep it terse, is it not clear?

Reply:
Thanks for the review @storopoli, good to see you again!